tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -1,2 +0,0 @@
-"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
```

Comment:
Also Commands.ResourceManager.Cmdlets is not included.  We should look at all our commands dlls and compare them to the list in SerializedCmdflets to ensure there are no other omissions.

Note: this is OK to merge as is, but we must add serializations for the additional assemblies ASAP.